components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationManagementService.java

```diff
@@ -339,6 +345,57 @@ public ApplicationResponseModel getApplication(String applicationId) {
         return new ServiceProviderToApiModel().apply(application);
     }
 
+    /**
+     * Get the authenticators configured for an application.
+     *
+     * @param applicationId ID of the application to be exported.
+     * @return  configured authenticators.
+     */
+    public ArrayList<ConfiguredAuthenticatorsModal> getConfiguredAuthenticators(String applicationId,
+                                                                                String tenantDomain) {
+
+        ArrayList<ConfiguredAuthenticatorsModal> response = new ArrayList<>();
+        try {
+            AuthenticationStep[] authenticationSteps = getApplicationManagementService()
+                    .getConfiguredAuthenticators(applicationId, tenantDomain);
```

Comment:
Since application is an UUID which is unique across the server do we need to pass tenant domain?